libp2p-swarm/src/swarm.rs

```diff
@@ -186,6 +188,7 @@ impl<T, C, H, If, F> Future for SwarmFuture<T, C, H, F>
                 self.to_process.push(future::Either::A(handler(connec, client_addr).into_future()));
             },
             Ok(Async::NotReady) => {},
+            // TODO: may not be the best idea because we're killing the whole server
             Err(err) => return Err(err),
```

Comment:
This behaviour should be configurable but I don't know how.

Reply:
Oh, this TODO is a pollution and shouldn't be in this PR.